package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "vite",
     "build:docs": "vite build --mode docs",
-    "build:lib": "vite build --mode lib && vue-tsc -p tsconfig.build.json",
+    "build:lib": "vite build --mode lib && pnpm run build-types",
+    "build-types": "vue-tsc -p tsconfig.build.json && api-extractor run",
```

Comment:
Can I change the script to `build:types`

Reply:
@xiaoluoboding done